packages/htmlbars-syntax/tests/parser-node-test.js

```diff
@@ -286,42 +286,42 @@ test("Node helpers", function() {
 test("Tokenizer: MustacheStatement encountered in tagName state", function() {
   var t = "<input{{bar}}>";
   astEqual(t, b.program([
-    b.element('input', [], [ b.sexpr(b.path('bar')) ])
+    b.element('input', [], [ b.mustache(b.path('bar')) ])
```

Comment:
Sorry, not sure if I get the idea. Do you want to group the `tag`, `attributes` and `helpers` property of `ElementNode` in a node called `ElementModifier`?